hls4ml/templates/quartus/firmware/nnet_utils/nnet_conv1d_resource.h

```diff
@@ -3,6 +3,7 @@
 
 #include "nnet_common.h"
 #include "nnet_dense.h"
+#include <cstdint>
```

Comment:
What do we use from this header?

Reply:
This header provides the `uint8_t` used in that file, and is not included by default in all platforms. Though, I am not sure if using `uint8_t` provides any performance benefit as that index should probably be optimized out when compiling to hdl...